### ISSUE TITLE
ab benchmarking utility won't work without '-k' option with Tempesta

### ISSUE THREAD

**Issue by keshonok** [natsys/tempesta#386]
Start `ab` against Tempesta with options to issue only 2 requests in one thread, and be very verbose:
```
ab -v 4 -n 2 -c 1 http://192.168.10.230:80/
```
It issues only one request and then times out after a while.

```
lta@bastinda:~/tfw-run/wget$ ab -v 4 -n 2 -c 1 http://192.168.10.230:80/
This is ApacheBench, Version 2.3 <$Revision: 1528965 $>
Copyright 1996 Adam Twiss, Zeus Technology Ltd, http://www.zeustech.net/
Licensed to The Apache Software Foundation, http://www.apache.org/

Benchmarking 192.168.10.230 (be patient)...INFO: GET header == 
---
GET / HTTP/1.0
Host: 192.168.10.230
User-Agent: ApacheBench/2.3
Accept: */*


---
LOG: header received:
HTTP/1.1 200 OK
Server: nginx/1.6.3
Date: Mon, 28 Dec 2015 17:52:44 GMT
Content-Type: text/html
Content-Length: 3700
Last-Modified: Sun, 04 Oct 2015 07:53:44 GMT
ETag: "5610db08-e74"
Accept-Ranges: bytes

<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
    <head>
        <title>Test Page for the Nginx HTTP Server on Fedora</title>
        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
        <style type="text/css">
            /*<![CDATA[*/
            body {
                background-color: #fff;
                color: #000;
                font-size: 0.9em;
                font-family: sans-serif,helvetica;
                margin: 0;
                padding: 0;
            }
            :link {
                color: #c00;
            }
            :visited {
                color: #c00;
            }
            a:hover {
                color: #f50;
            }
            h1 {
                text-align: center;
                margin: 0;
                padding: 0.6em 2em 0.4em;
                background-color: #294172;
                color: #fff;
                font-weight: normal;
                font-size: 1.75em;
                border-bottom: 2px solid #000;
            }
            h1 strong {
                font-weight: bold;
                font-size: 1.5em;
            }
            h2 {
                text-align: center;
                background-color: #3C6EB4;
                font-size: 1.1em;
                font-weight: bold;
                color: #fff;
                margin: 0;
                padding: 0.5em;
                border-bottom: 2px solid #294172;
            }
            hr {
                display: none;
            }
            .content {
                padding: 1em 5em;
            }
            .alert {
                border: 2px solid #000;
            }

            img {
                border: 2px solid #fff;
      
LOG: Response code = 200
apr_pollset_poll: The timeout specified has expired (70007)
```

**Comment by keshonok** [comment]
That happens because `ab` issues requests using HTTP/1.0 protocol. By default a connection should be closed after a response is sent out to the client. So if there's no `Connection:` header in an HTTP/1.0 request, that should be treated as if there's `Connection: close` in the request.

**Comment by keshonok** [comment]
Fixed in c59f98b.
